### PR TITLE
explicitly set serializer='pickle' for task, periodic_task, and analytics_task

### PR DESCRIPTION
**Pull request by nickpell**
This way, tasks that are passed non-JSON-serializable arguments will not fail in celery 4.x.

After the celery upgrade is complete, I'll encourage people to remove ```serializer='pickle'``` from tasks falling under their areas of ownership.

@emord Skipping ```old_analytics_task``` to avoid introducing merge conflicts with https://github.com/dimagi/commcare-hq/pull/21541, and because I figure that PR will be merged soon anyway.

This fixes this error on staging:
https://sentry.io/dimagi/commcarehq/issues/667077773/
which makes me additionally confident that this PR works.

### CODE REVIEW

File: corehq/apps/analytics/tasks.py [hunk 1]
```diff
@@ -247,7 +247,7 @@ def update_hubspot_properties(webuser, properties):
     update_hubspot_properties_v2(webuser, properties)
 
 
-@analytics_task()
+@analytics_task(serializer='pickle', )
```
Comment (emord) [inline corehq/apps/analytics/tasks.py]: Nit: it's weird to have the trailing comma in a decorator

also going back and forth between keeping this file the same and changing the util to keep pickling for now, but whichever you want to go with seems fine

**Review by emord — APPROVED** [review]
:heart: thanks for doing this! when do you think you'll change the default serializer to JSON? I think if this gets merged before that serializer change, then it leaves some time for new tasks to be added that could break

also looks like merging that PR caused a conflict :/

File: corehq/apps/analytics/tasks.py (hunk 1 above)
Comment (nickpell) [reply corehq/apps/analytics/tasks.py]: Frankly, if it being an eye-sore means somebody sooner will swap out the serializer for the default json one, then I'm cool with that.

Comment (nickpell) [reply corehq/apps/analytics/tasks.py]: I like defaulting the util to json, so new code gets written right the first time.

**Comment by nickpell** [comment]
@emord Yeah, it makes sense to default to JSON serialization now instead of waiting for celery 4.x

I updated settings.py.

**Comment by nickpell** [comment]
Holding off on merge; need to investigate https://sentry.io/dimagi/commcarehq/issues/667501683/

File: corehq/apps/analytics/tasks.py (hunk 1 above)
Comment (emord) [reply corehq/apps/analytics/tasks.py]: yeah. that makes sense. at first I was suprised that it worked, but I did a quick test locally and it seems fine

**Comment by nickpell** [comment]
Resolved by updating settings.  Could I get a review of the most recent changes?

**Comment by nickpell** [comment]
bump @emord @esoergel (sorry to nag.  I just really want to get this merged and get the celery 4.x work done so I can do the work I'm actually resourced on lol)

**Review by emord — APPROVED** [review]

**Pull request merged**